default_shardings/gemma.yaml

```diff
@@ -0,0 +1,20 @@
+
+# Sharding config for gemma
+# "replicated" to signify "replicated".
+# Integer signify axis to shard: 0 <= shard axis < rank
+
+freqs_cis : null #  torch.complex64 (16384, 128)
```

Comment:
Can we keep consistency on  replicated sharding? If either null or -1 is fine, shall we just keep -1 in our code base (use null in gemma, but -1 in llama)?